### PR TITLE
TRIVIAL: Update localizations

### PR DESCRIPTION
**Pull request by patrikbraborec**
---

Supported PR commands:

| Command                | Description            |
| ---------------------- | ---------------------- |
| `ok to test`           | Re-run standard checks |
| `extended test`        | BackstopJS tests       |
| `extended check sonar` | SonarQube tests        |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2891/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4520/ : SUCCESS in 16m 46s

File: libs/sdk-ui/src/base/localization/bundles/de-DE.json [hunk 1]
```diff
@@ -104,19 +104,29 @@
     "filters.nextNQuarters": "{n, plural, one {Nächstes # Quartal} other {Nächste # Quartale}}",
     "filters.nextNYears": "{n, plural, one {Nächstes # Jahr} other {Nächste # Jahre}}",
     "filters.interval.days.past": "Von {from} bis {to} {to, plural, one {Tag} other {Tage}} vorher",
+    "filters.interval.days.past.sameValue": "vor {value} {value, plural, one {Tag} other {Tagen}}",
```
Comment (petrjanu) [inline libs/sdk-ui/src/base/localization/bundles/de-DE.json:107]: `vor` (lower case) vs `Von`
@patrikbraborec , please make the first letter uppercase in the PR - applicable also for the strings below

**Review by petrjanu — CHANGES_REQUESTED** [review]

File: libs/sdk-ui/src/base/localization/bundles/de-DE.json (hunk 1 above)
Comment (petrjanu) [reply libs/sdk-ui/src/base/localization/bundles/de-DE.json:107]: We'll report the bug after

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2899/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4528/ : SUCCESS in 12m 29s

**Review by petrjanu — APPROVED** [review]

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/850/ : SUCCESS in 59s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1114/ : SUCCESS in 23s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4532/ : SUCCESS in 17m 59s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-el7/224/ : FAILURE in 21s (non-voting)

**Pull request merged**